swingset/src/main/java/com/nqadmin/swingset/SSBaseComboBox.java

```diff
@@ -563,6 +769,24 @@ public void setAllowNull(boolean _allowNull) {
 		adjustForNullItem();
 	}
 
+// TODO: Review this. GlazedLists is probably still calling it so need to detect those calls.
```

Comment:
I added then commented this out. Pretty sure GL choked on it. Probably have to add some code to ignore calls from GL, but would be useful in the future if someone is using a non GL combo and attempts setEditable(true).

Reply:
See Issue #46